skunkworks_crow/src/main/res/values/styles.xml

```diff
@@ -19,4 +19,30 @@
         <item name="android:tint">@android:color/white</item>
     </style>
 
+    <!-- Styles for Common Usage -->
+    <style name="mMatchParent">
```

Comment:
@huangyz0918 What does `m` in all the style signifies?

Reply:
Resolved.